### PR TITLE
feat/Missing e2e - added missing e2e data to select currency dropdown

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/CurrencySelect/CurrencySelect.js [hunk 1]
```diff
@@ -77,4 +78,4 @@ const mapDispatchToProps = dispatch => ({
   actions: bindActionCreators(actions.components.exchange, dispatch)
 })
 
-export default connect(getData, mapDispatchToProps)(CurrencySelect)
+export default connect(getData, mapDispatchToProps)(memo(CurrencySelect))
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/CurrencySelect/CurrencySelect.js:81]: Why is this necessary?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeForm/CurrencySelect/CurrencySelect.js:81]: I was testing something, ignore this, will revert. 
We have constant call of render of CurrencySelect on swap page, have to do some more digging why it occur like that
